modules/UI/videolayout/SmallVideo.js

```diff
@@ -557,24 +557,19 @@ SmallVideo.prototype._isHovered = function() {
  * Hides or shows the user's avatar.
  * This update assumes that large video had been updated and we will
  * reflect it on this small video.
- *
- * @param show whether we should show the avatar or not
- * video because there is no dominant speaker and no focused speaker
  */
 SmallVideo.prototype.updateView = function() {
     if (this.disableUpdateView) {
         return;
     }
 
-    if (!this.hasAvatar) {
```

Comment:
Why is this change? This doesn't seem to be related to LB mode

Reply:
It's in a separate commit. The screen-share icon looks tiny otherwise.